.github/workflows/ccpp.yml

```
@@ -204,3 +204,34 @@ jobs:
           -DDART_MSVC_DEFAULT_OPTIONS=ON ^
           -DDART_VERBOSE=ON
         cmake --build . --target ALL_BUILD -- /maxcpucount:4
+        
+  windows_2019_clang_cl:
+    name: Windows (clang-cl) [Release]
+    runs-on: windows-2019
+    env:
+      COMPILER: gcc
```

```suggestion
      COMPILER: clang
```
I know this wouldn't have any effect, but just to avoid some confusions

---

Done in https://github.com/dartsim/dart/pull/1509/commits/00459b64c6b1a0ad9098f9c5f378e34d0df2705b .